src/components/General/Layout.js

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
 import styled from 'styled-components';
+import BackgroundIcons from '../../backgroundIcons.svg';
 
-const StyledWrapper = styled.div`
-  max-width: 1170px;
-  margin: 0 auto;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
+const StyledMain = styled.main`
+  background-image: url(${BackgroundIcons});
+  min-height: 100vh;
```

Comment:
Sometimes `100vh` can come back to bite you (especially on mobile). Another way to accomplish this is to give 

height: 100% to `html` `body` and `#root` (the main div of the react app), and then have a `min-height: 100%` here.

Reply:
Sounds good. Thanks for the hot tip! Changes made 👍

Reply:
Sweet...I just learned this tip, myself.